### DISCUSSION TITLE
Sharpness settings in simple menu settings

### DISCUSSION THREAD

**Discussion by K1w3** [eduardofilo/RG350_adam_image#209] [SimpleMenu topics]
I notised that sharpness settings are gone from version 1.3 in simple menu settings. Boxart look a little pixelised from version 1.3.1 and above. Can you tell me where to find those settings?

**Comment by eduardofilo** [comment]
The sharpness settings were part of the release candidate version of SimpleMenu v11 installed in 1.3 which was removed in 1.3.1 and later.
Looking at the code of [that version of SM](https://github.com/fgl82/simplemenu/tree/experimental), I see that this setting is used to adjust the `SDL_VIDEO_KMSDRM_SCALING_SHARPNESS` environment variable that I assume will act on the SDL framework, but I don't know of an alternative system to this setting that was in SM and has now been lost. Hopefully, the SM developer will release these improvements in a while when he finishes that experimental version.

**Comment by K1w3** [comment]
Ok. Thank you for an explanation.
